source/_integrations/apsystems.markdown

```diff
@@ -0,0 +1,24 @@
+---
+title: APsystems
+description: Control and monitor your APsystems EZ1 microinverters locally without the cloud
+ha_iot_class: Local Polling
+ha_config_flow: true
+ha_release: 2024.6
+ha_category:
+  - Energy
+ha_domain: apsystems
+ha_platforms:
+  - sensor
+ha_integration_type: device
+---
+
+The APsystems {% term integration %} allows you to read the data from your EZ1 inverter.
```

Comment:
```suggestion
The APsystems {% term integration %} allows you to read the data from your [APsystems EZ1](https://emea.apsystems.com/diy/ez1/) microinverter.
```
I think it could be helpful to link to the official product page. Please correct the link if you have one in mind that is more suitable.

Reply:
The link is fine. Any other links would lead to shops, which is probably unwanted.